demos/calculators/style.css

```diff
@@ -0,0 +1,222 @@
+@import url("https://fonts.googleapis.com/css?family=Rubik:300,300i,400,400i,500,500i,700,700i,900,900i&display=swap");
+
+body {
+	font-family: Rubik, sans-serif;
+	font-weight: 300;
+	display: grid;
+	gap: 3rem 0;
+	justify-content: center;
+	max-width: 60vw;
+	margin-inline: auto;
+}
+
+header {
+	h1 {
+		font-size: 4rem;
+		font-weight: 300;
+		font-style: italic;
+		margin-block-end: 0;
+	}
+
+	p {
+		margin-block-start: 0;
+	};
+
+	.attribution {
+		font-weight: 500;
+		color: #ff0080;
+	}
+}
+
+main {
+	display: contents;
+}
+
+section {
+	font-size: 1.25rem;
+	font-weight: 300;
+
+	h2 {
+		font-size: 2em;
+		font-weight: 500;
+	}
+}
+
+a {
+	color: inherit;
+}
+
+input {
+	min-width: 4ch;
+	max-width: 10ch;
+	padding-block: .3em;
+	font-size: inherit;
+	border: none;
+	border-block-end: solid 1px;
+	text-align: center;
+}
+
+input:focus {
+	outline: none;
+}
+
+input:focus::-webkit-input-placeholder {
+	color: transparent !important;
+}
+
+input:focus::-moz-placeholder {
+	color: transparent !important;
+}
+
+input:focus:placeholder {
+	color: transparent !important;
+}
```

Comment:
Nit but I’d use nesting for these. Also why are we making the placeholder transparent? If it's to show a label, we can just show it on `:placeholder-shown` as I mentioned in the chat.

Reply:
> Also why are we making the placeholder transparent? If it's to show a label, we can just show it on `:placeholder-shown` as I mentioned in the chat.

According to the author's idea (as I get it), when the user focuses on an input field, the placeholder should disappear—the default behavior is that it's shown until something is typed. I think it's just for some aesthetic things.

Reply:
Okay, well it's poor for usability since you can't see what you're typing so let's remove it